mkt/reviewers/views.py

```diff
@@ -448,17 +444,12 @@ def app_review(request, addon):
         # them to index the app or call update_version() if that wasn't done
         # before already.
         if request.method == 'POST':
-            try:
+            with transaction.atomic():
```

Comment:
Thinking about this method more, do we *want* the whole transaction to be rolled back if there is an exception in the post_save handlers?  At that point (I think) we've already signed the zip, send the emails, etc.